### ISSUE TITLE
[ST] Relayer fee estimation issues

### ISSUE THREAD

**Issue by MinistroDolar** [connext/monorepo#6034]
**Describe the bug**
As shown in the screenshots, seems that we're having severe issues with relayer fee estimations (some of them are too high, some shown as Unknown)

<img width="374" alt="image" src="https://github.com/connext/monorepo/assets/26097731/3f696428-1c2e-4577-bb55-3d1ccabaa669">

<img width="389" alt="image" src="https://github.com/connext/monorepo/assets/26097731/88fb0904-61f8-4c20-9566-ab5bf159bf5c">

<img width="416" alt="image" src="https://github.com/connext/monorepo/assets/26097731/16510029-cf0a-4fae-8b04-f2582bc139f5">

<img width="380" alt="image" src="https://github.com/connext/monorepo/assets/26097731/d0f06c9e-a15a-499a-814e-1ac1ee395069">

<img width="401" alt="image" src="https://github.com/connext/monorepo/assets/26097731/141ef1a8-fc32-4123-b209-eaa6a812aa27">

**Link to ticket in discord**
https://discord.com/channels/454734546869551114/1228413683105796249

**Additional context**
@oncall

**Comment by preethamr** [comment]
I was able to repro this:
<img width="2422" alt="image" src="https://github.com/connext/monorepo/assets/1195648/0a9049c5-77d0-4a86-aec6-c2efa8bb031c">

**Comment by preethamr** [comment]
This is a DD log for the SDK-server request:
https://app.datadoghq.com/logs?query=host%3Aip-172-17-2-160.us-east-2.compute.internal%20service%3Asdk-server%20%40originChainId%3A137%20%40destinationChainId%3A10%20&cols=host%2Cservice%2C%40relayerFeeUsd&context_event=AY7UFeTyAABRs7wvDoSmagAU&event=AgAAAY7UFX8SEh2MQwAAAAAAAAAYAAAAAEFZN1VGWDlYQUFEX1drMmV1aldRbEFBRAAAACQAAAAAMDE4ZWQ0MTUtZDNjZS00ZjFiLTljZDgtNTM1YmMyN2RhZjRl&fromUser=true&index=%2A&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=time%2Cdesc&viz=&from_ts=1712954888021&to_ts=1712955189496&live=false

And the corresponding UI 
<img width="2000" alt="image" src="https://github.com/connext/monorepo/assets/1195648/97c0b93d-9806-48f8-b0e7-82dc9e119e80">
context:

**Comment by preethamr** [comment]
This is a UI issue, it gets a valid response from the sdk (sdk-server) and then shows `Unknown` to the user.
@regexpressyourself could you take a look please.

**Comment by regexpressyourself** [comment]
Will take a look right after #6041

**Comment by regexpressyourself** [comment]
Ok, looks like the `0` price data was due to coin gecko rate limiting. With coingecko working, we get proper gas estimation fees.

Going to keep trying to reproduce in case there's another issue at play, but I'm guessing the coingecko issues were the root cause.

**Comment by just-a-node** [comment]
This should be fixed now with private coingecko apis.
